### ISSUE TITLE
[Enhancement] 'Use Avatar' button should have some kind of feedback

### ISSUE THREAD

**Issue by misslivirose** [mozilla/hubs#1506]
Right now, there's no way that you can tell that your selection has been successful, unless you look down at yourself. Suggest some kind of visual or audio feedback.

**Comment by SparshKat** [comment]
Yes, i saw this issue and it's a grave issue. @brianpeiris @gfodor what do you guys suggest to do for this issue. I am willing to contribute to this too. 
One suggestion is that i can make some text come up when someone click the use avatar button just like whenever someone clicks the copy avatar button.

**Comment by gfodor** [comment]
I'm actually not sure what we want to do here. I don't think a text notification is the right approach, since we don't really have a place for it nor does that really convey the new avatar you've chosen is now your embodiment.

I think we probably should re-visit this issue if/when we decide how we want to enable users to see their own avatar. Currently you do it via opening the camera, which isn't really a great UX. Once that is decided upon, there's a good chance whatever mechanic we do here will need to incorporate it so the user can see the new avatar they've changed to using it.

**Comment by emclaren** [comment]
Personally, I would prefer a little bit of audio feedback, similar to what's used when you pin an object. It would be nice to have some indication that your selection was successful (particularly when you are select an avatar that has been placed in-world, as I never know if I clicked it correctly)

**Comment by gfodor** [comment]
That sounds like a good start! Would you like to give it a shot
@devilinside? You can just add the camera snapshot sound effect to start to
see how that feels.

**Comment by SparshKat** [comment]
Sure thing @gfodor , i'll try it out.

**Comment by SparshKat** [comment]
Could you once pin point to the file consisting of the 3d interactive VR file where i could dabble with changes with the avatar? @gfodor @emclaren

**Comment by SparshKat** [comment]
Sorry i wasn't able to find the correct file to submit a PR, a quick response would do the job :)

**Comment by johnshaughnessy** [comment]
The "use avatar" button is the open-media-button : 

https://github.com/mozilla/hubs/blob/master/src/components/open-media-button.js

**Comment by SparshKat** [comment]
<img width="1440" alt="Screenshot 2020-02-09 at 5 14 47 PM" src="https://user-images.githubusercontent.com/32457916/74101434-b5112f00-4b5f-11ea-9a0c-e65b4129a6db.png">
I can see no "Use Avatar" button anymore and the changing of avatar has been done by opening a link on the avatar. @johnshaughnessy  @gfodor how should i contribute to it now? :(
